src/components/explorer/ExplorerDatasetSearch.js

```diff
@@ -108,8 +108,8 @@ class ExplorerDatasetSearch extends Component {
         const numResults = (this.props.searchResults || {searchFormattedResults: []}).searchFormattedResults.length;
 
         const tableStyle = {
-            opacity: (this.props.fetchingSearch ? 0.5 : 1),
-            pointerEvents: (this.props.fetchingSearch ? "none" : "auto")
+            opacity: (this.props.fetchingSearch || this.props.fetchingTextSearch ? 0.5 : 1),
+            pointerEvents: (this.props.fetchingSearch || this.props.fetchingTextSearch ? "none" : "auto")
```

Comment:
`this.props.fetchingSearch || this.props.fetchingTextSearch` is reused a couple times; maybe it could be assigned a name above this definition to reuse later in `render()`